client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/index.tsx

```diff
@@ -44,60 +38,39 @@ const EmailProvidersStackedCard: FunctionComponent< ProviderCard > = ( props ) =
 	const toggleVisibility = ( event: React.MouseEvent ): void => {
 		event.preventDefault();
 
-		onExpandedChange( providerKey, ! detailsExpanded );
+		onExpandedChange( providerKey ?? '', ! detailsExpanded );
```

Comment:
Could you clarify when this `providerKey` prop wouldn't be available? As far I can see this is a static prop that is always provided.